src/wait.rs

```diff
@@ -148,16 +147,16 @@ impl SnapshotWaiter {
 
 /// Potential errors while waiting for the snapshot.
 mod error {
-    use rusoto_core::RusotoError;
-    use rusoto_ec2::DescribeSnapshotsError;
+    use aws_sdk_ec2::error::DescribeSnapshotsError;
     use snafu::Snafu;
 
     #[derive(Debug, Snafu)]
     #[snafu(visibility(pub(super)))]
     pub(super) enum Error {
         #[snafu(display("Failed to describe snapshots: {}", source))]
         DescribeSnapshots {
-            source: RusotoError<DescribeSnapshotsError>,
+            #[snafu(source(from(aws_sdk_ec2::types::SdkError<DescribeSnapshotsError>, Box::new)))]
+            source: Box<aws_sdk_ec2::types::SdkError<DescribeSnapshotsError>>,
```

Comment:
Is the box necessary? (Was it a clippy issue with the size of the error? If so a comment would be nice to explain why we boxed this one error type)

```suggestion
            source: aws_sdk_ec2::types::SdkError<DescribeSnapshotsError>,
```

Reply:
Yeah, it was a clippy size thing, I'll add a comment